src/main/java/de/tum/cit/aet/artemis/core/service/FileService.java

```diff
@@ -186,6 +186,32 @@ public URI handleSaveFile(MultipartFile file, boolean keepFilename, boolean mark
         return URI.create(markdown ? MARKDOWN_FILE_SUBPATH : DEFAULT_FILE_SUBPATH).resolve(currentFilename);
     }
 
+    /**
+     * Handles the saving of a file in a conversation.
+     *
+     * @param file           The file to be uploaded.
+     * @param courseId       The ID of the course.
+     * @param conversationId The ID of the conversation.
+     * @return The URI of the saved file.
+     */
+    public URI handleSaveFileInConversation(MultipartFile file, Long courseId, Long conversationId) {
+        // TODO: Improve the access check. The course is already checked, but the user might not be a member of the conversation. The course may not belong to the conversation
```

Comment:
Do you plan to add this check as part of this PR?

Reply:
We opted for a follow-up here :+1: